packages/core/src/utils/Util.ts

```diff
@@ -177,14 +177,75 @@ export const toScreen = (
 
 //#region color
 
-export const toHexRGB = (color: [number, number, number]): string => {
+export const hexToRgba = (
+  hex: string
+): [number, number, number, number] | null => {
```

Comment:
By our conventions from #873:
```suggestion
): [number, number, number, number] | undefined => {
```